README.md

```diff
@@ -248,6 +262,33 @@ To use no fallback when token authentication fails, set `fallback: :none`.
 
   [csrf]: https://github.com/gonzalo-bulnes/simple_token_authentication/issues/49
 
+### Testing with Rspec/Minitest
```

Comment:
Great! I'd write **Testing** instead of **Testing with ...**, but I can edit that when cherry-picking your commits.

Maybe both examples could be gathered in a single code block?

```ruby
class SomeControllerTest < ActionController::TestCase

  test "index token authentication can be performed via query params" do
    get :index, { user_email: "some@email.com", user_token: "some_token" }
    assert_response :success
  end

  test "index token authentication can be performed through request headers" do
    @request.headers['X-User-Email'] = "some@email.com"
    @request.headers['X-User-Token'] = "some_token"

    get :index
    assert_response :success
  end
end
```

I mean, even if you prefer to use one methods avor the other, in fact both are available anytime. What do you think?